### DISCUSSION TITLE
一分钟读论文：《工作良好的一天：软件工程师的日常》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#8] [Announcements]
# 一分钟读论文：《工作良好的一天：软件工程师的日常》

作为团队的 Leader，如何通过改进流程和⼯具，并最终提⾼团队⽣产⼒呢？可以参考微软研究院和英国伦敦大学、瑞士苏黎世大学信息学系合著的论文《Today was a Good Day: The Daily Life of Software Developers》。该论文分析了微软工程师的5,971份问卷结果，发现工程师「良好」和「典型」的工作日是怎么样，并总结了使良好的工作日成为典型的建议：

https://unbug.github.io/Today-Was-a-Good-Day-The-Daily-Life-of-Software-Developers/

**Comment by unbug** [comment]
需求规划阶段多拉研发开会，开发阶段少开会，多么简单有多少团队能做到呢？
